DigitalOffice/Services/CheckRightsService/Repositories/CheckRightsRepository.cs

```diff
@@ -23,5 +24,9 @@ public List<Right> GetRightsList()
         {
             return dbContext.Rights.Select(r => mapper.Map(r)).ToList();
         }
+
+        public bool CheckIfUserHaveRight(ICheckIfUserHaveRightRequest request)
+            => Enumerable.Any(dbContext.Rights.Where(right => right.Id == request.RightId),
```

Comment:
Hard to read

Reply:
I do not agree, this is a matter of habit. But the code didn't work, so I fixed it :)